CHANGES.md

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+- Fix the detection of the number of processors in a clean enviornment when
```

Comment:
What does it mean in a "clean environment"?

I would simplify to something like:

```
- Bootstrap: correctly detect the number of processors by allowing `nproc` to be looked up in `$PATH`.
```

Reply:
Thanks that's much better.